[2022-10-27T15:05:07.886+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: ml_pipeline.training_and_retraining_the_model.retraining_model manual__2022-10-27T15:04:57.310274+00:00 [queued]>
[2022-10-27T15:05:08.274+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: ml_pipeline.training_and_retraining_the_model.retraining_model manual__2022-10-27T15:04:57.310274+00:00 [queued]>
[2022-10-27T15:05:08.275+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-10-27T15:05:08.275+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-10-27T15:05:08.275+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-10-27T15:05:08.770+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): training_and_retraining_the_model.retraining_model> on 2022-10-27 15:04:57.310274+00:00
[2022-10-27T15:05:08.877+0000] {standard_task_runner.py:54} INFO - Started process 204 to run task
[2022-10-27T15:05:08.917+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'ml_pipeline', 'training_and_retraining_the_model.retraining_model', 'manual__2022-10-27T15:04:57.310274+00:00', '--job-id', '71', '--raw', '--subdir', 'DAGS_FOLDER/ml_pipeline.py', '--cfg-path', '/tmp/tmpuh834it7']
[2022-10-27T15:05:09.046+0000] {standard_task_runner.py:83} INFO - Job 71: Subtask training_and_retraining_the_model.retraining_model
[2022-10-27T15:05:09.048+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/ml_pipeline.py
[2022-10-27T15:05:15.806+0000] {local_task_job.py:87} ERROR - Received SIGTERM. Terminating subprocesses
[2022-10-27T15:05:15.809+0000] {process_utils.py:133} INFO - Sending Signals.SIGTERM to group 204. PIDs of all processes in the group: [204]
[2022-10-27T15:05:15.809+0000] {process_utils.py:84} INFO - Sending the signal Signals.SIGTERM to group 204
[2022-10-27T15:05:17.684+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=204, status='terminated', exitcode=<Negsignal.SIGTERM: -15>, started='15:05:08') (204) terminated with exit code Negsignal.SIGTERM
[2022-10-27T15:05:17.684+0000] {local_task_job.py:164} INFO - Task exited with return code 143
[2022-10-27T15:05:18.351+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
